queue_services/entity-filer/src/entity_filer/filing_processors/filing_components/correction.py

```diff
@@ -84,6 +85,16 @@ def correct_business_data(business: Business, correction_filing_rec: Filing,  #
         if business_start_date:
             business.start_date = datetime.datetime.fromisoformat(business_start_date) + timedelta(hours=8)
 
+    # update share structure and resolutions, if any
+    with suppress(IndexError, KeyError, TypeError):
+        share_structure = dpath.util.get(correction_filing, '/correction/shareStructure')
+        shares.update_share_structure(business, share_structure)
+    
+    # update provisionsRemoved, if any
+    with suppress(IndexError, KeyError, TypeError):
+        provisions_removed = dpath.util.get(correction_filing, '/correction/provisionsRemoved')
+        if provisions_removed is not None:
+            business.restriction_ind = not provisions_removed
```

Comment:
I am not sure whether `restriction_ind` is the correct attribute to update for “provisionsRemoved”, which is present in BEN IA corrections.

Reply:
I think this is the right field.